main/getting-started/intro-hackathon.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started for the Agoric Hackathon
```

Comment:
Let's title this Getting Started for HackTheOrb, The Agoric x Chainlink Hackathon!